iFixit/react-commerce
# Add micron genuine part badges

Adds Crucial and Micron svg components for use in the genuine partner badge on the product page.

Drops all SVG components in favor of direct imports via the `react-svgr` library as described [here](https://react-svgr.com/docs/next/).

## QA

Similar QA to https://github.com/iFixit/react-commerce/pull/918.

View a product that has the oem_partnership metafield set to `crucial` or `micron`, and verify that the badge has the same UI as the existing oem partnership badges.

Additionally, check that all other SVG components still work, since this pull changed the mechanism of how we import SVG's. Currently the only other SVG's are:
- the empty product list image (e.g. https://www.ifixit.com/Parts/Computer_Hardware)
- the empty search state image (e.g. https://www.ifixit.com/Parts?q=asdf)
- the quality guarantee image (e.g. bottom of https://www.ifixit.com/Parts or on product page of any product with a lifetime warranty)

CC @sterlinghirsh 

Closes https://github.com/iFixit/ifixit/issues/46144
